src/content/docs/en/reference/api-reference.mdx

```diff
@@ -397,18 +415,12 @@ if (!isLoggedIn(cookie)) {
 ---
 ```
 
-### `Astro.canonicalURL`
-
-:::caution[Deprecated]
-Use [`Astro.url`](#astrourl) to construct your own canonical URL. 
-:::
-
-The [canonical URL][canonical] of the current page.
-
```

Comment:
A note on this: we probably kept this around because there were some links to the `#astrocanonicalurl` heading slug. That said, I don’t think it’s still needed, because as you noted it was removed more than a year ago.

If we wanted to be responsible, we _could_ add a redirect for this case to `public/_redirects`:
```
/:lang:/reference/api-reference/#astrocanonicalurl   /:lang:/reference/api-reference/#astrourl
```

Reply:
The documentation doesn't seems to link to this section anymore, but external resources might so the redirection makes sense! I'll add it.